src/main/scala/core/MasterActor.scala

```diff
@@ -1,35 +1,30 @@
-package main
+package core
 
 import akka.actor.{Actor, ActorLogging, ActorRef, PoisonPill, Props, Terminated}
 import ch.qos.logback.classic.LoggerContext
 import ch.qos.logback.core.util.StatusPrinter
-import database.DatabaseActor
 import org.slf4j.LoggerFactory
-import scheduler.Basched
 import utils.Configuration
-import webserver.WebServerActor
 
 import scala.collection.mutable.ArrayBuffer
 
 /**
   * Created by igor on 10/05/17.
   */
-object MasterActor {
```

Comment:
It seems that the Akka team recommends declaring the `props` function in the [companion object](https://doc.akka.io/docs/akka/current/actors.html?language=scala#recommended-practices)

Reply:
sorry, make sense... will revert